frontend/src/app/products/item/item.component.html

```diff
@@ -1,22 +1,20 @@
 <mat-card class="product-card">
-  <img
-    class="preview"
-    [src]="defaultImage"
-    alt="Angular Logo"
-    (click)="goToDetails(product)"
-  />
+  <img class="preview" [src]="defaultImage" alt="Angular Logo" (click)="goToDetails(product)" />
   <section class="infos" (click)="goToDetails(product)">
     <header>
       <h3>
         {{ product.title }}
       </h3>
       <mat-chip-list>
-        <mat-chip *ngIf="product.status !== 'approved'">{{
+        <mat-chip *ngIf="product.status !== 'approved' && product.status !== 'rejected'">{{
```

Comment:
I think this doesn't make sense here. I'd advise to update the  ProductService in the backend